src/components/IconCard/icon-card.scss

```diff
@@ -1,22 +1,123 @@
-.icon-container {
+.icons-list {
   display: flex;
-  flex-flow: row wrap;
+  flex-flow: wrap;
   width: 100%;
   padding-bottom: $spacing-06;
 }
 
-.icon-search {
-  margin-bottom: rem(32px);
-}
-
-.icon {
-  flex: 1 1 10rem;
-  max-width: 10rem;
+.icon__container {
+  flex: 0 1 50%;
   position: relative;
 
   &:nth-child(n) {
+    border-top: 1px solid $carbon--gray-20;
     border-left: 1px solid $carbon--gray-20;
-    border-bottom: 1px solid $carbon--gray-20;
+  }
+  &:nth-child(n):nth-child(-n + 2) {
+    border-top: none;
+  }
+  &:nth-child(2n - 1) {
+    border-left: none;
+  }
+}
+
+@media all and (min-width: 515px) {
```

Comment:
Is there a reason we aren't using the Carbon media queries? can you add a comment saying why or if possible switch to the Carbon defaults?

Reply:
the component did have bespoke breakpoints before, and these are modified but based on the same reasoning. 

the breakpoints are hard coded and specific values to keep the icon tiles fitting properly (filling the full with of the container) at every possible window size